.github/workflows/pr.yml

```diff
@@ -27,37 +28,149 @@ jobs:
           filters: |
             has-files-requiring-all-checks:
               - "!(**.md|.github/CODEOWNERS)"
+
+  check-label:
+    runs-on: buildjet-2vcpu-ubuntu-2204
+    name: Check for E2E label
+    outputs:
+      run-e2e: ${{ steps.check-if-pr-has-label.outputs.run-e2e == 'true' && (github.event_name != 'labeled' || (github.event_name == 'labeled' && github.event.label.name == 'ready-for-e2e')) }}
+      run-jobs: ${{ github.event_name != 'labeled' }}
```

Comment:
Do we want to add the conditional for the `unlabeled` event as well?
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#label

Reply:
No, because the workflow isn't triggered at all for unlabeled

Reply:
Ah I see where now. This makes sense.